frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -68,28 +68,13 @@ const register = async (
       { firstName, lastName, email },
       { withCredentials: true },
     );
-    localStorage.setItem(AUTHENTICATED_USER_KEY, JSON.stringify(data));
+    // localStorage.setItem(AUTHENTICATED_USER_KEY, JSON.stringify(data));
```

Comment:
Will remove this comment later as we don't want to change local storage when a new user is registered.